From d31e32f2e46da2c2d9c72860959f6a5e305a595f Mon Sep 17 00:00:00 2001
From: Aetf <aetf@unlimited-code.works>
Date: Fri, 4 Feb 2022 13:39:20 -0500
Subject: [PATCH] tests: fix build for meson >= 0.61.0

`source_dir` is never used by `gnome.mkenums_simple`. Since commit
4d8c91134c08fc577e053ff007975e90c9bebf6d released in meson 0.61.0,
it rejects the argument rather than silently ignores it.
---
 tests/meson.build | 1 -
 1 file changed, 1 deletion(-)

diff --git a/tests/meson.build b/tests/meson.build
index e1eaedc..61ee43c 100644
--- a/tests/meson.build
+++ b/tests/meson.build
@@ -279,7 +279,6 @@ if get_option('tod')
     )
 
     fp_todv1_enums = gnome.mkenums_simple('fp-todv1-enums',
-        source_dir: 'tod-drivers',
         sources: [
             'tod-drivers/base-fp-device.h',
             'tod-drivers/base-fp-print.h',
-- 
2.35.1

